### PR TITLE
Add a section on common issues

### PR DESCRIPTION
**Pull request by Mohammed-Shoaib**
### Notes

* Create a new section on common issues and update the navigation page.
* Move the section on advice for asset files and audio stimuli to common issues.
* Small changes such as content tabs and typo.

### Testing

* Install [MkDocs](https://www.mkdocs.org/) by following the [documentation](https://www.mkdocs.org/user-guide/installation/).
* Use `mkdocs serve` and manually check the changes.

### CODE REVIEW

File: docs/docs/developer-guide/installation.md [hunk 1]
```diff
@@ -13,6 +13,22 @@ mkdir -p ~/roar-apps
...
+        ```sh
+        npm install tsup -D
+        ```
+
+    === "yarn"
+
+        ```sh
+        yarn add tsup --dev
+        ```
+
```
Comment (richford) [inline docs/docs/developer-guide/installation.md]: Not sure why we need `tsup` here.

**Review by richford — CHANGES_REQUESTED** [review]
@Mohammed-Shoaib, this looks good, thanks! I just have one correction which is that I'm not sure why we need `tsup`.

**Comment by Mohammed-Shoaib** [comment]
Thank you for the review, Adam!

I got the an error when running `npm start`:

```
tsc: command not found
```

I had typescript installed. Installing `tsup` fixed the issue for me.

Do you have `tsup` installed on your machine?

**Review by richford — APPROVED** [review]

**Comment by Mohammed-Shoaib** [comment]
Updated to remove the instructions on `tsup`!

**Pull request merged**